### PR TITLE
Function creating a tile #3

### PR DESCRIPTION
**Pull request by Kesse8401**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const tileOnClickHandler = () => console.log('tile number');
+                                                                    
+
+const square = () => {
+    const tile = document.createElement('div');
+    tile.addEventListener( 'click', tileOnClickHandler );
+    tile.className = 'cipka';
+ 	return tile;
```
Comment (bartlomn) [inline index.js]: Indent to tile word above

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const tileOnClickHandler = () => console.log('tile number');
+                                                                    
+
+const square = () => {
+    const tile = document.createElement('div');
+    tile.addEventListener( 'click', tileOnClickHandler );
+    tile.className = 'cipka';
```
Comment (bartlomn) [inline index.js]: Change the classname to something decent - this repo is public

File: style.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.cipka {
```
Comment (bartlomn) [inline style.css]: Change the classname to something decent - this repo is public

**Review by bartlomn — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Kesse8401) [reply index.js]: It does looks differently (correctly) in my text editor for some reason

**Review by bartlomn — APPROVED** [review]
Looking good, please merge.
Remember to pull your local master branch after.

**Pull request merged**